### ISSUE TITLE
Missing pot footprints: 9mm rotary and Bourns PTA slide

### ISSUE THREAD

**Issue by evanshultz** [KiCad/Potentiometers.pretty#19]
When looking at https://github.com/KiCad/Potentiometers.pretty/pull/11, I realized that while we have a good number of pot footprints there are still quite a few not in the library. Or we have footprints for only a subset of the parts in a pot family.

Here is the list of members I can find in each family, with ones I find in the current library in a grey box. I grouped them together even if shaft length changes, but for completeness each one would need a unique footprint since the 3D model would have to be different.

[ALPS RK09L](http://www.alps.com/prod/info/E/HTML/Potentiometer/RotaryPotentiometers/RK09L/RK09L_list.html):
- `Single Horizontal` (shaft lengths: 15, 20mm)
- `Single Vertical` (shaft lengths: 12.5, 15, 20, 25mm)
- `Dual Horizontal` (shaft lengths: 15, 25mm)
- `Dual Vertical` (shaft lengths: 15, 20mm)

[ALPS RK097](http://www.alps.com/prod/info/E/HTML/Potentiometer/RotaryPotentiometers/RK097/RK097_list.html):
- `Single Horizontal` (shaft lengths: 15, 20, 25mm)
- `Dual Horizontal` (shaft lengths: 15, 20, 25mm)
- Single Horizontal + Switch or Momentary 0.5mm (shaft lengths: 15, 20mm)
- Dual Horizontal + Switch or Momentary 0.5mm (shaft lengths: 15, 20mm)
- Single Horizontal + Push On/Off (shaft lengths: 25mm)
- Dual Horizontal + Push On/Off (shaft lengths: 20mm)
- Single Horizontal + Momentary 1.5mm (shaft lengths: 20mm)
- Single Horizontal + Push Lock (shaft lengths: 25mm)
- Dual Horizontal + Push Lock (shaft lengths: 25mm)
- Dual Horizontal Dual Shaft (shaft lengths: 15/25mm)

[ALPS RK09K/RK09D](http://www.alps.com/prod/info/E/HTML/Potentiometer/RotaryPotentiometers/RK09K/RK09K_list.html):
- `Single Vertical`  (shaft lengths: 15, 20, 25, 30mm)
- `Single Horizontal 6.5mm`  (shaft lengths: 15, 20mm)
- `Single Horizontal 10mm`  (shaft lengths: 15, 20mm)
- Single Vertical + Collar  (shaft lengths: 20, 25, 30mm)
- Single Horizontal 10mm + Collar  (shaft lengths: 25mm)
- Dual Vertical  (shaft lengths: 15, 20, 30mm)
- Dual Vertical + Collar  (shaft lengths: 25, 30mm)
- Dual Horizontal 6.5mm  (shaft lengths: 15, 20mm)
- Dual Horizontal 6.5mm + Collar  (shaft lengths: 20mm)

[ALPA RK09Y](http://www.alps.com/prod/info/E/HTML/Potentiometer/RotaryPotentiometers/RK09Y11/RK09Y11_list.html):
- `Single Horizontal` (shaft lengths: 12mm)

ALPS RV09:
I don't find a product page on ALPS' website, but searching online shows a number of pots marked as "RV09" which seem to fit into the footprints above.

[Bourns PTA](http://www.bourns.com/docs/Product-Datasheets/pta.pdf):
- PTA1543
- PTA2043
- PTA3043
- PTA4543
- `PTA6043` (unfinished from the PR above)
- PTA1544
- PTA2044
- PTA3044
- PTA4544
- PTA6044

[Bourns PTV09](http://www.bourns.com/docs/Product-Datasheets/ptv09.pdf):
- `PTV09A-1`
- `PTV09A-2`
- PTV09A-3
- PTV09A-4
- PTV09A-5
- PTV09A-6
- PTV09A-7
- PTV09A-8

@andrew8er @jkriege2 
Thanks for your work to step in and contribute to the aforementioned PR! Are either of you willing to help fill in the gaps? I can also contribute to this body of work.

Note that at least some of the footprints in the library mentioned above are called "Vertical" when they're really horizontal, and vice versa. See https://github.com/KiCad/Potentiometers.pretty/issues/18. It may be best to correct this mis-naming first since it will only further confuse things. Note that I've used the correct naming above, even if it conflicts with the library footprint names.

**Comment by andrew8er** [comment]
I'm currently not working on those, but feel free to rename and reuse everything I contributed here. Would be nice if all footprints where properly named and organised.

**Comment by secondaryObject** [comment]
Hi, I'm new to the community so I'm not sure how things works here. Anyway I was searching for Bourns PTA slider and I found some here: 

https://github.com/mcous/kicad-lib/tree/master/footprints/bourns.pretty

I could not open them but I was able to copy important parts into another kicad mod file. 
My question is:

In case I create those PTA footprints shall I push them into the repo or is there some requirements to have also 3D model. I installed KiCAD today so just trying to explore and contribute a little.
